File: exp/exp.py

# ATOMIC_FUNCTION:  exp(val)
# TAGS: math，numeric
# LANG:  python
# FUNCTION: exp(val)
import math

start_val = math.log(1)
expected_end_val = 1

end_val = math.exp(start_val)


# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")

File: exp/exp.php
<?php
// ATOMIC_FUNCTION:  exp($val)
// TAGS: math，numeric
// LANG:  php
// FUNCTION: exp($val)

$start_val = log(1);
$expected_end_val = 1;

$end_val = exp($start_val);

// TEST
if($end_val == $expected_end_val) {
	echo "PASS";
} else {
	
	echo "FAIL";
}

?>
